golang-bionic/analyzer/pkg/restrictpkg/testdata/src/ignore/ignore.go

```diff
@@ -0,0 +1,29 @@
+package a
```

Comment:
The package name should be `ignore`, shouldn't it?

Reply:
I think it should be `main`. Because it has a main function.